src/plugins/components/typography/wysiwyg.js

```diff
@@ -13,7 +13,6 @@ module.exports = function () {
           marginBottom: '0',
         },
         'p, ol, ul, dl': {
-          color: theme('color.black.true'),
```

Comment:
I don't feel like we need to dictate what text color people use in a WYSIWYG. Original I picked the true black (#000) thinking it would improve a11y since the contrast is higher, but Kerri mentioned that it's not always the case. Default for HTML document is black text anyhow so I think it's ok for us to remove this.